libs/appstore/src/appstore/zomes/portal-zome-client.ts

```diff
@@ -15,23 +15,12 @@ export class PortalZomeClient extends ZomeClient {
   }
 
   async customRemoteCall<T>(input: CustomRemoteCallInput): Promise<T> {
-    const response = await this.callZome('custom_remote_call', input);
```

Comment:
Oh, was this the part that you had to change to not get error "undefined"?

Reply:
Yes